modules/date.js

```diff
@@ -0,0 +1,12 @@
+import { DateTime } from './luxon.min.js';
+
+const dateTime = document.getElementById('dateTime');
+
+const setInitialTime = () => {
+  const dt = DateTime.now();
+  const getCurrentTime = () => `${dt.hour}:${dt.minute}:${dt.second}`;
+  const f = { month: 'long', day: 'numeric' };
+  dateTime.innerHTML = `${dt.setLocale('en-US').toLocaleString(f)} ${dt.year}, <span>${getCurrentTime()}</span>`;
+};
+
+export default setInitialTime;
```

Comment:
- [ ] Nice work so far I see you have implemented the data functionality using 'Luxon' but you are currently not displaying you date anywhere and as per project requirements you are expected  to disp you date and time below the navbar on your header section 👍🏿